Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardItemRow.swift

```diff
@@ -78,17 +76,14 @@ struct CreditCardItemRow: View {
             .padding(.trailing, 16)
             .padding(.top, 11)
             .padding(.bottom, 11)
-            .background(isTapped ? backgroundHoverColor : backgroundColor)
-            .gesture(
-                TapGesture()
-                    .onEnded { _ in
-                        didSelectAction?()
-                    }
-                    .simultaneously(with: DragGesture(minimumDistance: 0))
-                    .updating($isTapped) { (_, isTapped, _) in
-                        isTapped = true
-                    }
-            )
+            .background(isTapping ? backgroundHoverColor : backgroundColor)
+            .onTapGesture {
+                isTapping = true
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                    isTapping = false
```

Comment:
```suggestion
```
See my explanation below for the removal of isTapping = false

https://github.com/mozilla-mobile/firefox-ios/pull/15335/files?diff=unified&w=0#r1255165876

Reply:
removing isTapping = false would make sense for the CreditCardBottomSheet, but that cell is also used in the Credit Cards Autofill screen, and removing that line would cause your cells to remain selected when going back to the screen after editing anything.

Reply:
Yes, should have clarified that its for the case of bottom sheet @petruSt24

Reply:
@nbhasin2 so you basically want that cell to know if it's used by the bottom sheet or the settings screen? That would introduce another unneeded layer of complexity and that line of code would still remain there but under an if screen is Autofill, so it would look even worse.

Reply:
So much for getting the correct UX ha

Leave it as is then. R+ on this change